Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: build publish
+.PHONY: build publish docs docs-clean docs-serve
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)